tests/std/tests/P0896R4_P1614R2_comparisons/test.cpp

```diff
@@ -254,12 +254,13 @@ constexpr bool test_compare_three_way() {
 
 enum class some_enum { value };
 
-#if defined(__clang__) || defined(__EDG__) // TRANSITION, VSO-905257
 STATIC_ASSERT(test_compare_three_way<int, int, strong_ordering>());
 STATIC_ASSERT(test_compare_three_way<int, long, strong_ordering>());
 STATIC_ASSERT(test_compare_three_way<float, float, partial_ordering>());
 STATIC_ASSERT(test_compare_three_way<float, double, partial_ordering>());
+#if defined(__clang__) || defined(__EDG__) // TRANSITION, GH-741
 STATIC_ASSERT(test_compare_three_way<long, double, partial_ordering>());
+#endif // TRANSITION, GH-741

```

Comment:
Filed Devcom-1044530 "long <=> double works, but not in a concept":
```suggestion
#if defined(__clang__) || defined(__EDG__) // TRANSITION, DevCom-1044530
STATIC_ASSERT(test_compare_three_way<long, double, partial_ordering>());
#endif // TRANSITION, DevCom-1044530
```